cms/static/cms/js/require-config.js

```diff
@@ -50,8 +50,8 @@
             'jquery.immediateDescendents': 'coffee/src/jquery.immediateDescendents',
             'datepair': 'js/vendor/timepicker/datepair',
             'date': 'js/vendor/date',
-            'moment': 'js/vendor/moment.min',
-            'moment-with-locales': 'js/vendor/moment-with-locales.min',
+            moment: 'common/js/vendor/moment-with-locales',
```

Comment:
Flagging because it seems odd, might not actually be a problem - why were the `'` marks removed here?